### DISCUSSION TITLE
Can't login through github

### DISCUSSION THREAD

**Discussion by zapbra** [vercel/community#3113] [Help] [answered]
### Summary

So, I'm not exactly sure what's going on, but I swear I used to log in to my vercel account through my github, but now when I try to sign in through github it says, "There is no Vercel account linked with this GitHub account."

I am at least able to login through my email, but I feel like my github, which has projects hosted on vercel should be attached to my vercel account, no?

I just tried to sign up for a new vercel account with the email "lesbianhaircut@outlook.com" and it logged me into my vercel account. Then I logged out and back in and it worked. Then I logged back in and I got the same error message.

I also had an issue yesterday with my deployments being queued, which I don't really care, but this may be related.

Let me know if you have any questions, thanks.
![verc](https://github.com/vercel/community/assets/84364656/cf902afd-7393-465c-8cf9-d16058c804cd)
![verc2](https://github.com/vercel/community/assets/84364656/7c1fa8db-20c3-4945-8c46-50d76c50f87e)
![verc3](https://github.com/vercel/community/assets/84364656/25ff5c2a-652c-4778-83b6-fc498f09a439)
![verc4](https://github.com/vercel/community/assets/84364656/38331f33-468a-4a7b-8fe0-3f02c54531b4)

### Steps to Reproduce

Try to login through github, get an error "There is no Vercel account linked with this GitHub account".
 Try to create a new account or login through email verification and it works.

**Comment by amyegan** [comment]
I checked and it looks like your GitHub username is currently linked to an account with the email address you shared. Are you still having login trouble?

**Reply by Pemond-296** [reply]
I have a same problem , can you check it for me? my Github username is Pemond-296

**Reply by leandroauzier** [reply]
I also not able to connect by doing this

**Reply by tranviet0710** [reply]
@amyegan i have the same problem, can you check it for me? I have already e-mailed to registration@vercel.com.

**Reply by amyegan** [reply]
If you already opened a new case and have a ticket number, please let me know the number so I can check on the open case for you 🙏 

If you haven't already done it, please open a ticket by using the appropriate [direct channel](https://vercel.com/guides/how-to-get-vercel-support#direct-channels) found in this guide: [What is the best way to get support from Vercel?](https://vercel.com/guides/how-to-get-vercel-support#direct-channels)

**Reply by 3mperorsSeal** [reply]
hi there my ticket number is  00166597. any help would be much appreciated. Thanks.

**Reply by mrpitch** [reply]
Hi, same for me: ticket number 00166643. Thanks

**Reply by tranviet0710** [reply]
@amyegan  My ticket is 00165136. Please help me. Thank you so much.

**Reply by superen7527** [reply]
@amyegan amyegan hi there my ticket number is 00168337. any help would be much appreciated. Thanks.

**Comment by munan56** [comment]
Login with github, I get the error
There is already an account associated with your GitHub email address. Please login with your email and password then link your GitHub account. [Login with Email](https://vercel.com/login/email?next=%252Faccount%252Flogin-connections)

Then Login with email, I get the error

We are unable to complete your signup. Please read [this guide](https://vercel.com/guides/why-can-i-not-signup) for further details.

**Reply by coverai-co** [reply]
I have the same issue

did you ever solve this? @munan56 @amyegan

**Reply by munan56** [reply]
No response from anyone 😂

**Comment by Bonitas-Group** [comment]
Hello @amyegan  I'm having the same issue as @munan56 I cant login with GitHub, Email and I can not create a new account with my email address or GitHub account.

**Comment by swarnava** [comment, marked as answer]
For anyone who are still having trouble to login / signup, please email us at registration@vercel.com. Unfortunately, we cannot review registration related issue via this channel. Thank you!
